tardis/montecarlo/base.py

```diff
@@ -31,6 +32,7 @@
 
 MAX_SEED_VAL = 2**32 - 1
 
+
```

Comment:
extra whitespace?

Reply:
@andrewfullard Yeah Black formatter was adding this when I saved the file. Overriden that behaviour and removed this now.